src/components/table/td.ts

```diff
@@ -28,6 +28,10 @@ const RW_OPTIONS = [
 
 const R_OPTIONS = [{ label: 'Copy', value: 'copy' }]
 
+if (customElements.get('astra-td')) {
+  // System has already registered `astra-td`
+}
+
```

Comment:
this shouldn't be necessary and the warnings should only be triggered in development? is this to suppress the warnings or is it actually broken?

Reply:
Yeah this throws errors on hot reloading in react (the big red dialog covering the screen), and also in the terminal repeatedly when used. This is to attempt it from being re-added multiple times to the registry.

My thinking here is that if it's added to the registry, there's no reason for it to be re-added to the registry again. If you have a better idea for this I'm all ears though – as long as it rids my logs and screens of pesky noise :)

Reply:
I commented too hastily without reading the links you provided. Makes sense that JS includes it only a single time but the React runtime seems to do it again on hot reloading. Ultimately whatever can clean up the terminal & prevent the red screen though is my ultimate goal so whatever is a sound solution I'm all for!